Editor/EditorCore/EditorMeshUtility.cs

```diff
@@ -19,6 +20,17 @@ public static class EditorMeshUtility
         [UserSetting("Mesh Editing", "Auto Resize Colliders", "Automatically resize colliders with mesh bounds as you edit.")]
         static Pref<bool> s_AutoResizeCollisions = new Pref<bool>("editor.autoRecalculateCollisions", false, SettingsScope.Project);
 
+        [UserSetting("Mesh Settings", "Removed Unused Materials", "Automatically remove materials that are no longer used.")]
```

Comment:
`Auto Remove Material` was a better naming, I would not use "Removed" in the settings name.